### PR TITLE
Add keyboard shortcut to toggle collapse

### PR DESCRIPTION
**Pull request by bsell93**
Hello! 👋🏻

I really like this extension!

I thought it would be nifty to add a shortcut to toggle collapse to make it easier to open/close.

Do with this what you will. Just thought I'd make a PR if you are interested in having this.

I also thought of adding to your "options" page the ability to change the shortcut value, but haven't done that yet.

Thanks for this awesome extension!
Bryant

PS. If you'd like other changes or potentially other features would love to help if I can 😄

### CODE REVIEW

**Comment by jdhayford** [comment]
Hey @bsell93 thanks for the pull request! I built your branch and tried it out. Definitely like the idea of adding this functionality to the extension.

The only change I might ask for before we merge this would be to see if you can keep the `Collapse/Expand` button in sync even when using the shortcut. Currently, it does it by checking if any of the tabs groups are collapsed, but it only checks when the popup is initially mounted.


https://user-images.githubusercontent.com/9739680/122956722-40532500-d34f-11eb-9811-850a64c893e8.mp4


I'd be open to any solution on that, including putting another check on an interval as long as it's short enough to feel responsive without having any noticeable effect on performance.

PS: Bonus points if we can get a clean tooltip on that button to hint to users that they can use the shortcut :)

**Comment by bsell93** [comment]
@jdhayford Thanks for the feedback!

With the latest changes I've made the popup button text update appropriately as well as added desired tooltip! 😄 

Let me know what you think!

File: src/pages/Popup/RuleForm.jsx
Comment (jdhayford) [inline src/pages/Popup/RuleForm.jsx:356]: 🙌🏼

**Comment by jdhayford** [comment]
@bsell93 Works like a charm, nice job! I'm going to merge it. Will try to bump the version and release it tonight.

Btw if you want to chat about features or any other changes feel free to ping me on twitter (https://twitter.com/hayford_jack).

**Review by jdhayford — APPROVED** [review]

**Pull request merged**
